internal/p2p/transport_test.go

```diff
@@ -599,7 +599,7 @@ func dialAccept(ctx context.Context, t *testing.T, a, b p2p.Transport) (p2p.Conn
 	endpoints := b.Endpoints()
 	require.NotEmpty(t, endpoints, "peer not listening on any endpoints")
 
-	ctx, cancel := context.WithTimeout(ctx, time.Second)
+	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
Anything we can say here about how long this should be?  E.g.,

```go
// Leave the endpoints enough time to be reachable even if startup is slow.
```

or whatever may make sense?